src/runner/nunit.runner.Droid/Resources/Values/Strings.xml

```diff
@@ -1,5 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<resources>
-    <string name="ApplicationName">$projectname$</string>
-    <string name="Hello">Hello</string>
```

Comment:
I added this file to fix https://github.com/nunit/nunit.xamarin/issues/57. Is it no longer needed?

Reply:
I guess not, if we have nothing platforms specific any more?

Reply:
The android test project is building and running fine with the latest and as you say, this isn't an Android library anymore 😄 

Good eye though.